kitchen-tests/kitchen.dokken.yml

```diff
@@ -14,37 +13,39 @@ provisioner:
     always_dump_stacktrace: true
   chef_license: "accept-no-persist"
+  chef_license_key: free-58e5ca65-d1cc-4838-bb79-42aa40ed3623-3835
   deprecations_as_errors: true
   slow_resource_report: true
   clean_dokken_sandbox: false
 
 lifecycle:
   pre_converge:
-    - remote: echo "Chef container's Chef / Ohai release:"
-    - remote: /opt/chef/bin/chef-client -v
-    - remote: /opt/chef/bin/ohai -v
-    - remote: echo "Installing appbundler and appbundle-updater gems:"
-    - remote: /opt/chef/embedded/bin/gem install appbundler appbundle-updater --no-doc
-    # back compat for pre-unified-/usr distros, do not add new OSes
-    - remote: sudo ln -s /usr/bin/install /bin/install
-      includes:
-        - opensuse-leap-15
-        - debian-11
-    - remote: sudo yum install centos-release-scl
-      includes:
-        - centos-7
-    - remote: scl enable devtoolset-7 '/opt/chef/embedded/bin/appbundle-updater chef chef <%= ENV['GITHUB_SHA']  || %x(git rev-parse HEAD).chomp %> --tarball --github <%= ENV['GITHUB_REPOSITORY']  || "chef/chef" %>'
-      includes:
-        - centos-7
-        - oraclelinux-7
-    - remote: echo "Updating Chef using appbundler-updater:"
-    - remote: /opt/chef/embedded/bin/appbundle-updater chef chef <%= ENV['GITHUB_SHA']  || %x(git rev-parse HEAD).chomp %> --tarball --github <%= ENV['GITHUB_REPOSITORY']  || "chef/chef" %>
-      excludes:
-        - centos-7
-        - oraclelinux-7
-    - remote: echo "Installed Chef / Ohai release:"
-    - remote: /opt/chef/bin/chef-client -v
-    - remote: /opt/chef/bin/ohai -v
+    #appbundle-updater changes will happen once it is hab compatible
+    # - remote: echo "Chef container's Chef / Ohai release:"
+    # - remote: /opt/chef/bin/chef-client -v
+    # - remote: /opt/chef/bin/ohai -v
+    # - remote: echo "Installing appbundler and appbundle-updater gems:"
+    # - remote: /opt/chef/embedded/bin/gem install appbundler appbundle-updater --no-doc
+    # # back compat for pre-unified-/usr distros, do not add new OSes
+    # - remote: sudo ln -s /usr/bin/install /bin/install
+    #   includes:
+    #     - opensuse-leap-15
+    #     - debian-11
+    # - remote: sudo yum install centos-release-scl
+    #   includes:
+    #     - centos-7
+    # - remote: scl enable devtoolset-7 '/opt/chef/embedded/bin/appbundle-updater chef chef <%= ENV['GITHUB_SHA']  || %x(git rev-parse HEAD).chomp %> --tarball --github <%= ENV['GITHUB_REPOSITORY']  || "chef/chef" %>'
+    #   includes:
+    #     - centos-7
+    #     - oraclelinux-7
+    # - remote: echo "Updating Chef using appbundler-updater:"
+    # - remote: /opt/chef/embedded/bin/appbundle-updater chef chef <%= ENV['GITHUB_SHA']  || %x(git rev-parse HEAD).chomp %> --tarball --github <%= ENV['GITHUB_REPOSITORY']  || "chef/chef" %>
+    #   excludes:
+    #     - centos-7
+    #     - oraclelinux-7
+    # - remote: echo "Installed Chef / Ohai release:"
+    # - remote: /opt/chef/bin/chef-client -v
+    # - remote: /opt/chef/bin/ohai -v
 
```

Comment:
This is all needed to update the chef-client to the latest git commit sha version in the PR so any changes they are making to code are tested. Otherwise code changes wont be tested till they are merged an a new build is created and published to unstable channel and then it'll break all PR's if bad code was merged and built.

Reply:
@Stromweld  Yes, we are already working on appbundle-updater (mentioned in the comment) which has changes to make it habitat packaging compliant https://github.com/chef/appbundle-updater/pull/107 is the PR if you want to take a look. A new version of gem will be released once that PR is merged, which will be pulled in here